src/analysis/feature_set.jl

```diff
@@ -0,0 +1,111 @@
+"""
+    _filter_constants(scens::DataFrame)::DataFrame
+
+Filter out features/factors that do not vary.
+"""
+function _filter_constants(scens::DataFrame)::DataFrame
+    varying_cols = []
+    for (i, col) in enumerate(eachcol(scens))
+        if !all(col .== col[1])
+            push!(varying_cols, i)
+        end
+    end
+
+    return scens[:, varying_cols]
+end
+
+"""
+    _seeded(rs::ResultSet)::Tuple{Vector}
+
+Extract total and average deployment, currently in terms of proportional increase to cover
+relative to the locations' carrying capacity.
+
+# Arguments
+- `rs` : ResultSet holding scenario outcomes
+
+# Returns
+DataFrames of mean and total deployment for each coral group
+"""
+function _seeded(rs::ResultSet)::Tuple{DataFrame,DataFrame}
+    deployed_corals = rs.seed_log.coral_id
+
+    mean_deployment = [
+        mean(sum(rs.seed_log[:, c_id, :, :]; dims=:timesteps); dims=:locations).data[:]
+        for c_id in deployed_corals
+    ]
+
+    total_deployment = [
+        sum(sum(rs.seed_log[:, c_id, :, :]; dims=:timesteps); dims=:locations).data[:]
+        for c_id in deployed_corals
+    ]
```

Comment:
Note: this assumes `Yseed` in `scenario.jl` has been updated to log approximate number of seeded corals, not the relative increase in coral cover.

This change will be in a separate PR.